test/stress_tests.py

```diff
@@ -108,5 +108,145 @@ def g(x):
         self.assertTrue(ray.services.all_processes_alive())
         ray.worker.cleanup()
 
+class ReconstructionTests(unittest.TestCase):
+
+  def setUp(self):
+    # Start a Redis instance and a Plasma store instance with 1GB memory.
+    node_ip_address = "127.0.0.1"
+    redis_address = ray.services.start_redis(node_ip_address)
+    self.plasma_store_memory = 10 ** 9
+    plasma_address = ray.services.start_objstore(node_ip_address,
+                                                 redis_address,
+                                                 objstore_memory=self.plasma_store_memory)
+    address_info = {
+        "redis_address": redis_address,
+        "object_store_addresses": [plasma_address],
+        }
+
+    # Start the rest of the services in the Ray cluster.
+    ray.worker._init(address_info=address_info, start_ray_local=True,
+                     num_workers=1)
+
+  def tearDown(self):
+    # Clean up the Ray cluster.
+    self.assertTrue(ray.services.all_processes_alive())
+    ray.worker.cleanup()
+
+  def testSingleNodeSimple(self):
+    # Define the size of one task's return argument so that the combined sum of
+    # all objects' sizes is twice the plasma store's allotted memory.
+    num_objects = 1000
+    size = int(self.plasma_store_memory * 2 / (num_objects * 8))
```

Comment:
Since this is in `stress_tests.py`, can we make it `self.plasma_store_memory * 10` instead of `* 2`? (same with the other tests)